src/Versioned.php

```diff
@@ -2874,11 +2874,14 @@ public function isModifiedOnDraft()
      */
     public static function get_including_deleted($class, $filter = "", $sort = "")
     {
-        $list = DataList::create($class)
-            ->where($filter)
-            ->sort($sort)
-            ->setDataQueryParam("Versioned.mode", "latest_versions");
-
+        $list = DataList::create($class);
+        if (!empty($filter)) {
+            $list = $list->where($filter);
```

Comment:
Can we use `filter` in line with the other work related to replacing where with filter?

```suggestion
            $list = $list->filter($filter);
```

Reply:
That should be done as part of this issue - https://github.com/silverstripe/silverstripe-framework/issues/10601